src/main/java/ch/uzh/ifi/hase/soprafs22/service/LobbyService.java

```diff
@@ -242,4 +243,19 @@ public Collection<ILobby> getLobbiesCollection() {
         return lobbyManager.getLobbiesCollection();
     }
 
+    public Player addPlayer(String invitationCode, Long lobbyId) {
+        ILobby lobby = lobbyManager.getLobbyWithId(lobbyId);
```

Comment:
you can use the method getLobbyByIdElseThrowNotFound in the lobbyService class to get the lobby and else throw an exception

Reply:
oh ya that's true, thanks! mine does throw an exception but I did it additionally